### DISCUSSION TITLE
Is the Servo performance as fast as Blink?

### DISCUSSION THREAD

**Discussion by Saad-2023** [servo/servo#29257] [Q&A] [answered]
Is the Servo performance as fast as Blink(Chromium)? Overall is faster or slower?

**Comment by jdm** [comment, marked as answer]
It is slower because it has had less effort devoted to it.

**Reply by arjpar** [reply]
Why is it that Blink is still faster than Servo despite Servo using all these nice (CS) research concepts? I guess it goes to show how important testing and optimization is? (I understand that Servo is also still in development pre-stable, which is a huge reason too).

**Reply by Saarsk** [reply]
> hy is it that Blink is still faster than Servo despite Servo using all these nice (CS) research concepts? I guess it goes to show how important testing and optimizatio

This is completely normal for software in early development. :) Even mature software commonly have potential for further optimizations.

**Comment by KDEFTW-web** [comment]
On the up side, it loaded servo.org faster than Brave (could be a cache thing? or that the page has been optimised for for Servo?) google.com about the same speed, debian.org same speed, though after being loaded, Servo loads it faster from cache than Brave.

**Reply by KDEFTW-web** [reply]
though when reloading servo.org without clearing cache, Servo is still faster
